### PR TITLE
Feat/export win details

### PR DESCRIPTION
**Pull request by peterhudec**
## Description of change

Adds the _Export Wins Details_ page and updates links to it.

## Test instructions

1. Go to `/exportwins`
2. Click on on the _Review export win_ button
3. That should navigate to `/exportwins/<id>/details`
4. The page should conform to the design
5. There should be an _Export wins_ link on the bottom of the page, which should get you back to `/exportwins`

## Screenshots

https://github.com/uktrade/data-hub-frontend/assets/2333157/8959e8b0-a530-4807-9375-2b64980a5a0a

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/middleware/api-proxy.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ const ALLOWLIST = [
   '/v3/search/order',
   '/v3/event/',
   '/v4/event/',
-  '/v4/export-win/',
-  '/v4/export_win',
+  '/v4/export_win/',
+  '/v4/export_win/:exportWinId',
```
[the remark this answers is not in the document]
Comment (peterhudec) [reply src/middleware/api-proxy.js:60]: This is dictated by [`data-hub-api`](https://github.com/uktrade/data-hub-api/blob/05aa853acec9f85b8fec62008f1b892e14f57fc0/datahub/export_win/urls.py#L15-L18).

File: src/client/modules/ExportWins/Details.jsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        <ExportWin id={winId} progressBox={true}>
+          {(exportWin) => {
+            // TODO: Same / similar computation is used in the add export win form so it should be reused
+            const {
+              groupedBreakdowns = [],
+              totalAmount,
+              yearRange,
+              totalYears = yearRange?.max - yearRange?.min + 1,
+            } = exportWin?.breakdowns.reduce(
+              ({ groupedBreakdowns, totalAmount, yearRange }, x) => ({
```
Comment (DeanElliott96) [inline src/client/modules/ExportWins/Details.jsx:69]: What is x? Makes it hard for me to tell what it is without context of the object.

File: test/component/cypress/specs/ExportWins/Details.cy.jsx [hunk 3]
```diff
@@ -1,40 +0,0 @@
-import React from 'react'
```
Comment (DeanElliott96) [inline test/component/cypress/specs/ExportWins/Details.cy.jsx:1]: Are you going to add new tests for the details or do they already exist?

**Review by DeanElliott96 — COMMENTED** [review]
From your video, it looks good. I'm having some issues trying to run export wins on my frontend and have requested access so can't test myself atm.

Just a couple questions on the tests and the variable `x`.

File: src/middleware/api-proxy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dredmonds) [reply src/middleware/api-proxy.js:60]: I echo this issue to DH-API dev now is on-going for amendment.

**Review by dredmonds — APPROVED** [review]

File: test/component/cypress/specs/ExportWins/Details.cy.jsx (hunk 3 above)
Comment (peterhudec) [reply test/component/cypress/specs/ExportWins/Details.cy.jsx:1]: Yeah, this was just a stub for tests. The thing is now fully tested.

**Pull request merged**